integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java

```diff
@@ -139,7 +140,8 @@ public synchronized void allocateContainer(Container container) {
     mOutstandingContainerRequestsLatch.countDown();
   }
 
-  private void requestContainers(int numContainersToRequest) throws Exception {
+  private void requestContainers() throws Exception {
+    int numContainersToRequest = mTargetNumContainers - mAllocatedContainers.size();
```

Comment:
Yea, I just noticed that. My test passed the first time but subsequent standup failed. I missed that getPotentialWorkerHosts() method. I'll move numContainersToRequest and the LOG message below the if/else and resubmit.